Update files-in-native-mode.md (#1000)
MicrosoftDocs/OfficeDocs-O365ITPro: yammer/configure-your-yammer-network/files-in-native-mode.md

- Non-SharePoint group files will deleted *within 30 days* after the migration is completed.
- For any group containing multiple files with the same name, the duplicate-named files will be appended with _X, where X is an increasing number for each duplicate named file (for example, file_1 file_2 file_3 and so on).

> #### File renaming rules:

Reviewer: ```suggestion
## File renaming rules

```

- Characters not supported in SharePoint will be replaced with an underscore '_'.
- Duplicate files or those with names that already exist in SharePoint will be renamed using the following format: filename_yammerFileID_extension.
- Files with a blank space as the first or last character as well as files that end with a period will have those characters removed.

> - Unnamed files will be named according to the following format: "Yammer File", "Yammer File (2)", "Yammer File (3)", etc.
> - Files with names starting with \~$ will have the leading tilde removed, eg: "~$Yammer File" will be renamed to "$Yammer File".
> - Files with names containing _vti_ anywhere in the name will have it replaced with - vti-, eg: "Yammer_vti_File" will be renamed to "Yammer-vti-File".
> - Files named .lock, CON, PRN, AUX, NUL, COM0 - COM9, LPTO - LPT9, or desktop.ini will have "__file" appended to the name, eg: "COM0" will be renamed to "COM0_file". 

Reviewer: ```suggestion
- Unnamed files will be named according to the following format: "Yammer File", "Yammer File (2)", "Yammer File (3)", and so on.
- Files with names starting with \~$ will have the leading tilde removed, for example: "~$Yammer File" will be renamed to "$Yammer File".
- Files with names containing _vti_ anywhere in the name will have it replaced with - vti-, for example: "Yammer_vti_File" will be renamed to "Yammer-vti-File".
- Files named .lock, CON, PRN, AUX, NUL, COM0 - COM9, LPTO - LPT9, or desktop.ini will have "__file" appended to the name, for example: "COM0" will be renamed to "COM0_file". 
```
